.github/workflows/filtered_github_webhooks.yml

```diff
@@ -0,0 +1,33 @@
+# Filters GitHub webhooks to avoid noise in the Discord server.
+name: Execute Filtered GitHub Webhook
+
+on:
+  pull_request:
+    types:
+      - opened
+      - closed
+      - reopened
+
+jobs:
+  notify_if_not_dependabot:
```

Comment:
this is super pedantic of me, but what do you think about giving this a `name: Notify Discord` or something similar so it shows up all nice in the Actions UI?